.github/workflows/monorepo-docker.yml

```diff
@@ -10,7 +10,6 @@ on:
       - 'typescript/infra/**'
 concurrency:
   group: build-push-monorepo-${{ github.ref }}
-  cancel-in-progress: true
```

Comment:
I thought that was actually desirable in the cases where we keep pushing commits on a PR, we don't really need to comlpete jobs of previous commits or is this a different situation?